spec/inferno/dsl/test_creation_spec.rb

```diff
@@ -434,6 +437,28 @@
         end
       end
     end
+
+    describe 'empty_group' do
```

Comment:
This is test runner behavior, not test creation behavior, so it should be in the test runner spec.

Reply:
You're right. I kept the 'contains zero tests' spec in anyways as a negative test, but let me know if I should move this too